rainforest/environments.go

```diff
@@ -16,9 +18,14 @@ type Environment struct {
 
 // CreateTemporaryEnvironment creates a new temporary environment and returns the
 // Environment.
-func (c *Client) CreateTemporaryEnvironment(urlString string) (*Environment, error) {
+func (c *Client) CreateTemporaryEnvironment(runDescription string, urlString string) (*Environment, error) {
+	name := "temporary-env-for-custom-url-via-CLI"
+	if runDescription != "" {
+		name = fmt.Sprintf("%v-temporary-env", runDescription)
```

Comment:
Do we have any internal limits on the length of an environment name?

Reply:
We do, great catch 👍🏼